mantis/cli/stitch.py

```diff
@@ -0,0 +1,197 @@
+import datetime
+import shutil
+
+from pathlib import Path
+
+import click
+import numpy as np
+import pandas as pd
+
+from iohub import open_ome_zarr
+from iohub.ngff_meta import TransformationMeta
+from slurmkit import SlurmParams, slurm_function, submit_function
+
+from mantis.analysis.AnalysisSettings import StitchSettings
+from mantis.analysis.stitch import (
+    get_grid_rows_cols,
+    get_image_shift,
+    get_stitch_output_shape,
+    preprocess_and_shift,
+    stitch_shifted_store,
+)
+from mantis.cli.parsing import config_filepath, input_position_dirpaths, output_dirpath
+from mantis.cli.utils import create_empty_hcs_zarr, process_single_position_v2, yaml_to_model
+
+
+@click.command()
+@input_position_dirpaths()
+@output_dirpath()
+@config_filepath()
+@click.option(
+    "--temp-path",
+    type=click.Path(exists=True, file_okay=False, dir_okay=True),
+    default='/hpc/scratch/group.comp.micro/',
+    help="Path to temporary directory, ideally with fast read/write speeds.",
+)
+@click.option("--slurm", "-s", is_flag=True, help="Run stitching on SLURM")
```

Comment:
```suggestion
```
I'll suggest deleting this and the `if not slurm` block, since this option isn't really optional.